funnel/assets/Makefile

```diff
@@ -6,3 +6,12 @@ assets:
 
 assetsonly:
 	npm run build
+
+source_dir = ../translations
+target_dir = ../static/translations
+baseframe_dir := $(shell python ../../manage.py baseframe_translations_path)
+
+babel:
+	@mkdir -p $(target_dir)
+	ls $(source_dir) | egrep '[[:lower:]]{2}_[[:upper:]]{2}' | xargs -I % sh -c 'mkdir -p $(target_dir)/% && ./node_modules/.bin/po2json --pretty $(source_dir)/%/LC_MESSAGES/messages.po $(target_dir)/%/messages.json'
```

Comment:
macOS has BSD `grep` by default, with different regex syntax. Is this why you're using `egrep` here?

Reply:
`grep` doesn't seem to support the `{2}` part. Hence used `egrep` here.

Reply:
`grep -e` or `grep -E` should do that, but the default BSD grep on macOS may not support it.